readme.md

```diff
@@ -110,6 +110,7 @@
 - [Web Accessibility](https://github.com/brunopulis/awesome-a11y)
 - [Material Design](https://github.com/sachin1092/awesome-material)
 - [CSS](https://github.com/sota0805/awesome-css)
+- [Gulp](https://github.com/alferov/awesome-gulp)
```

Comment:
Gulp isn't specific to front end development. Not sure where it deserves though.